Org profile readme. (#1)
runziggurat/.github: profile/README.md

> <p align="center">

Reviewer: ```suggestion
# Ziggurat 

<p align="center">
```

    <img src="assets/logo.png" height="300px">
</p>
<br />
Ziggurat is a collection of network testing tools and a suite of conformance, performance and resistance tests our team developed in Rust.

# About

The project is developed by [Equilibrium](https://equilibrium.co) that provides the target networks with the foundation of reliability, performance, and scalability needed to realize the true potential of the network. Our clients  include, but not limited to, ZCash, Algorand, and XRPL Ledger.

> # Covergage

Reviewer 2: ```suggestion
# Coverage
```

> |       | **Spec Conformance** | **Fuzzing** | **Crawler** | **Chaos + Malicious Actor Modeling** | **Visualization** | **Performance Benchmarking (loosely defined)** |

Reviewer: ```suggestion
|       | **Spec Conformance** | **Fuzzing** | **Crawler** | ** Malicious Actor Modeling** | **Visualization** | **Performance Benchmarking ** |
```

|--------------------|--------|-------------|----------|-------------|-----------------|--------|

> | ZCash              | Yes (de-facto)  | Yes    | No          | Kinda    | No          | Yes    |

Reviewer: ```suggestion
| ZCash              | Yes  | Yes    | In Progress | Yes          | In Progress          | Yes    |
```

> | XRPL Ledger        | In progress     | No     | No          | No       | No          | No     |

Reviewer: ```suggestion
| XRPL Ledger        | In progress     | In Progress     | No          | In Progress       | In Progress          | In Progress     |
```

Author: Update Xrpl.

> | Algorand           | No              | No     | No          | No       | No          | No     |

Reviewer: ```suggestion
| Algorand           | In Progress              | No     | No          | No       | No          | No     |
```

| Aleo               | No              | No     | In Progress | No       | In Progress | No     |  
| Polkadot           | No              | No     | No          | No       | No          | No     |

> | Solana             | No              | No     | No          | No       | No          | No     |

Reviewer: ```suggestion
```

> | Eth2 Beacon Chain  | No              | No     | No          | No       | No          | No     |

Reviewer: ```suggestion
```

# Contributing

> Feedback and Contributions from the open source community are welcome. If you have any query, you can open an issue in relevant repository and our Team will try to take it from there.

Reviewer: ```suggestion
Feedback and Contributions from the open source community are welcome. If you have any queries, you can open an issue in the relevant repository and our team will take it from there.
```

<p align="center">
    <img src="assets/logo.png" height="300px">
</p>
<br />
Ziggurat is a collection of network testing tools and a suite of conformance, performance and resistance tests our team developed in Rust.

# About

> The project is developed by [Equilibrium](https://equilibrium.co) that provides [zcashd](https://github.com/zcash/zcash) and [zebra](https://github.com/ZcashFoundation/zebra) developers with the foundation of reliability, performance, and scalability needed to realize the true potential of the Zcash network.

Reviewer 2: We should make this generic—we're not only testing Zcash these days 😉

Reviewer: The full list is ZCash, Algorand, and XRPL Ledger. Hopefully Polkadot soon too 🤞

# FOMO Table
[TODO *Some description of the Table*]
|          | **Fuzzing** | **Crawler** | **Chaos + Malicious Actor Modeling** | **Visualization** | **Spec Conformance** | **Performance Benchmarking (loosely defined)** |
|--------------------|--------|-------------|----------|-------------|-----------------|--------|
| ZCash              | Yes    | No          | Kinda    | No          | Yes (de-facto)  | Yes    |
| Algorand           | No     | No          | No       | No          | No              | No     |
| Aleo               | No     | In Progress | No       | In Progress | No              | No     |  
| Polkadot           | No     | No          | No       | No          | No              | No     |
| Cardano            | No     | No          | No       | No          | No              | No     |
| Solana             | No     | No          | No       | No          | No              | No     |
| Mina               | No     | No          | No       | No          | No              | No     |
| Eth2 Beacon Chain  | No     | No          | No       | No          | No              | No     |

> | Pathfinder         | No     | No          | No       | No          | No              | No     |

Reviewer: Let's remove Cardano, Pathfinder, and Mina from the list _for now_

# Contributing [*Or something else*]

> [TODO]

Reviewer: For now just say that issues and PRs are accepted
